lib/generators/module/templates/Gemfile.erb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
```

Comment:
I don't think we need this line if these Engines are going to remain part of the `vets-api` codebase